requirements.txt

```
@@ -1,4 +1,4 @@
 requests==2.32.4
 loguru==0.7.3
-remarkable-update-image==1.2; sys_platform != 'linux'
+remarkable-update-image==1.2
```

```suggestion
remarkable-update-image==1.2; sys_platform != 'linux'
```
This is intentional as remarkable-update-fuse pulls remarkable-update-image already. All this change does it potentially break remarkable-update-fuse being included if it depends on a different version of remarkable-update-image.